pkg/controller/importconfig/manager.go

```diff
@@ -123,6 +123,19 @@ func Add(ctx context.Context,
 					UpdateFunc:  func(e event.UpdateEvent) bool { return true },
 				})),
 		).
+		Watches(
+			&corev1.Secret{},
```

Comment:
will this watch all secrets in the hub cluster?

Reply:
Good point, change to watch only secrets under pod namespace.